README.md

```diff
@@ -22,26 +30,78 @@ browzine.script.src = "https://s3.amazonaws.com/browzine-adapters/summon/browzin
 document.head.appendChild(browzine.script);
 ```
 
+_BrowZine 360 Core Adapter Script_
+
+```
+var browzine = {
+  api: "https://api.thirdiron.com/public/v1/libraries/XXX",
+  apiKey: "ENTER API KEY",
+};
+
+browzine.script = document.createElement("script");
+browzine.script.src = "https://s3.amazonaws.com/browzine-adapters/360-core/browzine-360-core-adapter.js";
+document.head.appendChild(browzine.script);
+```
+
+_Loading Discovery Service Adapters_
+
+```
+var browzine = {
+  api: "https://api.thirdiron.com/public/v1/libraries/XXX",
+  apiKey: "ENTER API KEY",
+};
+
+browzine.script = document.createElement("script");
+browzine.script.src = "https://s3.amazonaws.com/browzine-adapters/summon/browzine-summon-adapter.js";
+document.head.appendChild(browzine.script);
+
+browzine.script = document.createElement("script");
+browzine.script.src = "https://s3.amazonaws.com/browzine-adapters/360-core/browzine-360-core-adapter.js";
+document.head.appendChild(browzine.script);
+```
+
+_Adding Summon Custom Script_
+
 Summon Custom Scripts are added in the Summon 2.0 External Script portion of the Summon Editor Settings:
 ![Summon 2.0 Editor Link](https://i.imgur.com/HJFpDGm.png "Summon 2.0 Editor Link")
 
 As a Custom Script Url:
 ![Summon 2.0 External Script](https://i.imgur.com/piLMSic.png "Summon 2.0 External Script")
 
+
+_Adding 360 Core Custom Script_
```

Comment:
It might be a little easier for people new to this repo (like myself!) to follow if we group the instructions together by product.  So organization like this:

* Summon Adapter Installation
  * BrowZine Summon Adapter Script
  * Adding Summon Custom Script
  * Customizations
* 360 Core Adapter Installation
  * BrowZine 360 Core Adapter Script
  * Adding 360 Core Custom Script
  * Customizations

If you'd like me to reorganize it, let me know!

Reply:
OMG, YES! HEHE. See, the "Loading Multiple Discovery Service Adapters" note above too:
https://github.com/thirdiron/browzine-summon-adapter/pull/15#discussion_r162673571

Thanks @karlbecker!